utils/.coveragerc

```diff
@@ -0,0 +1,6 @@
+[run]
+omit=
+    */lib/python*/site-packages/*
+    */encodings/ascii.py
+    */getpass.py
+    */contextlib.py
```

Comment:
They are pulled in as dependencies, the first two lines exclude items that were explicitly installed within the virtualenv...  I'm not sure where getpass.py and contextlib.py are coming in from, but I'll see if I can figure it out.